percona_playback/query_log/query_log.cc

```diff
@@ -156,7 +161,8 @@ void* ParseQueryLogFunc::operator() (void*)  {
 	  next_len= len;
 	  break;
 	}
-	tmp_entry->add_query_line(std::string(line));
+        if((is_ro_mode && (strncmp("select", line, strlen("select")) == 0 || strncmp("SELECT", line, strlen("SELECT")) == 0)) || !is_ro_mode) 
```

Comment:
Why not use strnicmp() here instead of two compares?  why do we need to do two strlen() calls for a static string?